boefjes/tests/test_nmap.py

```diff
@@ -174,7 +174,7 @@ def test_normalizer(self):
         boefje_meta.arguments["input"] = serialize_ooi(input_ooi)
         output = list(run(get_normalizer_meta(boefje_meta), get_dummy_data("raw/nmap_mispoes.xml")))
         self.assertEqual(17, len(output))
-        for i in range(0, len(output)):
+        for i in range(0, len(output) - 1):
```

Comment:
Using `enumerate` would even be better. You'll never get out of bounds and you get a tuple with the index and value for each iteration

Reply:
Because it's usually writing less code and making the code more runtime and memory-efficient. But also in your case, it is to prevent getting out of bounds (I assume that was your intention based on the commit message)

Reply:
Have you looked at the subsequent code? I sincerely fail to see how `enumarate` would prevent an out-of-bounds?
Also the difference will be 8 characters.

Reply:
Yes, I've carefully reviewed the subsequent code. It's true that the use of 'enumerate' has minimal impact on character count, but the primary focus here is on enhancing code clarity and preventing potential out-of-bounds issues. Dealing with not accessing the last element can generally be done using slices. I'll leave it up to you